### ISSUE TITLE
Add PyPi Published Key to cookiecutter.json

### ISSUE THREAD

**Issue by lafemmephile** [lafemmephile/cookiecutter-hypy#1]
If code will not be published to `PyPI` but will instead stay hosted on `GitHub`, need a way to declare that. Could use a *pypi_published* key in `cookiecutter.json`

Then use control flow:
```jinja
{%- if cookiecutter.pypi_published -%}
# In case of True 
pip install {{ cookiecutter.package_name }} 

{%- else -%}
# In case of False 
pip install 'git+https://github.com/{{ cookiecutter.github_user }}/{{ cookiecutter.project_name }}.git'

{% endif %}
```

**Comment by lafemmephile** [comment]
Additional References:
+ https://cookiecutter.readthedocs.io/en/stable/advanced/boolean_variables.html

**Comment by lafemmephile** [comment]
Implemented with: 0774da785c984fc58d538cd66a48c2964f5f35b7
